salt/tests/unit/modules/test_metalk8s_network.py

```diff
@@ -0,0 +1,87 @@
+from salt.exceptions import CommandExecutionError
+
+from salttesting.mixins import LoaderModuleMockMixin
+from salttesting.unit import TestCase
+from salttesting.mock import MagicMock, patch
+
+import metalk8s_network
+
+
+class Metalk8sNetworkTestCase(TestCase, LoaderModuleMockMixin):
+    """
+    TestCase for `metalk8s_network` module
+    """
+    loader_module = metalk8s_network
+    loader_module_globals = {
+        "__pillar__": {
+            "networks": {
+                "service": "10.0.0.0/8"
+            }
+        }
+    }
+
+    def test_get_kubernetes_service_ip_success(self):
```

Comment:
There's quite a bit of duplication between tests for both methods. Can we parametrize test cases with the proposed approach?

Reply:
With pytest `paramtrize` nop but with `parameterized` yes sure, but here I don't see any benefits it's 2 differents function we test here even if the result are the same parametrized do not look really usefull, we could define some variable for error message but that pretty much it to me